apps/prairielearn/src/components/AssessmentRegenerate.html.ts

```diff
@@ -27,13 +27,12 @@ export function RegenerateInstanceModal({ csrfToken }: { csrfToken: string }) {
 
 export function RegenerateInstanceAlert() {
   return html`
-    <div class="alert alert-warning alert-dismissible fade show">
-      You are viewing your own assessment instance and have instructor permissions.
+    <div class="alert alert-warning border-warning alert-dismissible fade show">
+      Course staff:
       <a href="#" role="button" data-toggle="modal" data-target="#regenerateInstanceModal">
```

Comment:
Nit: should this be a button with `btn-link`?

Reply:
Those don't show up well when used inline with text, IIRC.